packages/components/src/TimeFormatter/TimeFormatter.mdx

```diff
@@ -0,0 +1,33 @@
+---
+name: TimeFormatter
+menu: Components
+route: /components/time-formatter
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { TimeFormatter } from ".";
+
+# TimeFormatter
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+In Jobber a TimeFormatter is used to …
+
+## Simple Example
+
+<Playground>
+  <TimeFormatter civilTime={"11:30:00.000000000"} useMilitaryTime={false} />
```

Comment:
I'm not sure that `MilitaryTime` is a broad name to use here.

Reply:
Perhaps `useAnteMeridiem` or use `use12HourClock`? I feel like there is a more conventional name here I'm just not thinking of it…